src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -190,8 +194,9 @@ export default {
     },
   },
   watch: {
-    isFocused(newVal) {
-      if (newVal) {
+    async isFocused(newVal) {
+      await waitFrames(8);
+      if (newVal && this.isRendered && this.enableSelfFocus) {
```

Comment:
This is to prevent a component that is outside of the viewport to get focused. This may happen if you scroll down and it goes out of the visible viewport.